app/Models/User.php

```diff
@@ -34,4 +37,20 @@ class User extends Model {
      * @var bool
      */
     public $timestamps = true;
+
+    public function getEmailForPasswordReset()
+    {
+        return $this->email;
+    }
+
+    /**
+     * Send the password reset notification.
+     *
+     * @param  string  $token
+     * @return void
+     */
+    public function sendPasswordResetNotification($token)
+    {
+        $this->notify(new ResetPasswordNotification($token));
+    }
```

Comment:
you can use  `Illuminate\Auth\Passwords\CanResetPassword` trait instead, and by extending `Illuminate\Foundation\Auth\User` you don't need to use `CanResetPassword` because it's implemented by default